src/test/regress/multi_schedule

```diff
@@ -140,7 +140,7 @@ test: multi_create_insert_proxy
 test: multi_data_types
 test: multi_repartition_udt
 test: multi_repartitioned_subquery_udf
-test: multi_modifying_xacts
+#test: multi_modifying_xacts
```

Comment:
Let's re-enable that, and just disable the offending test, and add an issue to re-enable that individual task.